### PR TITLE
Squash loops together.

### PR DESCRIPTION
**Pull request by 7415963987456321**
This commit squashes the loops rendering the lines together, now the first loop does both the top and bottom line, and the second loop does both left and right. 

EDIT: I screwed up with git and ended up putting what was supposed to be two seperate pull requests into this one. Anyways this also fixes the input issue in #4 as per the 400136e commit.

### CODE REVIEW

**Comment by francma** [comment]
I've looked into that parsing problem and it's caused by `scanf` allowing integer overflow. For example for input `256` it will produce `1`. By changing type to `uint16_t` it will be `1` for `65537` and so on. I have fixed it in https://github.com/francma/wob/pull/6.

Can you please edit this PR so it will contain only the "Squash loops together" commit?

File: wob.c [hunk 1]
```diff
@@ -256,7 +256,7 @@ main(int argc, char **argv)
 		return EXIT_FAILURE;
 	}
 
-	uint32_t * argb = wob_create_argb_buffer(&app), i;
+	uint32_t * argb = wob_create_argb_buffer(&app), i,k;
```
Comment (francma) [inline wob.c]: Add space after comma.

```suggestion
	uint32_t * argb = wob_create_argb_buffer(&app), i, k;
```

File: wob.c [hunk 2]
```diff
@@ -265,50 +265,38 @@ main(int argc, char **argv)
...
 		for (int pixel = 0; pixel < WIDTH - 2*BORDER_OFFSET; ++pixel) {
 			argb[i++] = WHITE;
+			argb[k++] = WHITE;
 		}
 		i += BORDER_OFFSET;
+		k += BORDER_OFFSET;
 	}
 
-	// create left horizontal line
+	// create right horizontal line
```
Comment (francma) [inline wob.c]:
```suggestion
	// create right and left vertical line
```

File: wob.c (hunk 1 above)
Comment (7415963987456321) [reply wob.c]: Fixed.

File: wob.c (hunk 2 above)
Comment (7415963987456321) [reply wob.c]: Fixed.

**Comment by 7415963987456321** [comment]
All requested changes fixed.

**Pull request merged**